tools/phenomenal/isatools/converters/isatab2json.xml

```diff
@@ -1,37 +1,50 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <tool id="isatab2json" name="isatab2json" version="1.0">
-  <description>Converts an ISA-Tab to ISA-JSON</description>
-  <command><![CDATA[
-    run_tab2json.py $isatab.extra_files_path
-  ]]>
-  </command>
-  <inputs>
-      <param name="isatab" label="ISA" type="data" format="isa-tab"/>
-  </inputs>
-  <outputs>
-    <data format="json" name="output" from_work_dir="out.json" label="ISA content in ISA-JSON format" />
-  </outputs>
-
-  <help>
+    <description>Converts an ISA-Tab to ISA-JSON</description>
+    <requirements>
+        <requirement type="package" version="6.7">click</requirement>
```

Comment:
Are these requirements for completeness ? Do we honour them in phnmnl at all, where the tool is encapsulated in container ?

Reply:
As far as I know the `<requirements>` section is ignored in our PhenoMeNal Galaxy that runs the tools in containers. But I think the section should be kept if it is declared in the vanilla Galaxy tool. There's no harm in keeping it in, and we are trying to keep the different versions of our Galaxy tools (vanilla vs. protein "flavoured") as closely aligned as possible.

Reply:
Exactly the responses I expected. I will merge now, hoping that this does not break the other PR in flight.